<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewerPage</class>
 <widget class="QWidget" name="PreviewerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>651</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="m_wndSplitterVert">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="m_wndSplitterHorz">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="m_modelViewportFrame" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="ReflectedPropertiesPanel" name="m_tagsPanel" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="m_wndSplitterTracks">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>3</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="CMannNodesWidget" name="m_wndNodes">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>300</height>
        </size>
       </property>
      </widget>
      <widget class="CMannDopeSheet" name="m_wndTrackPanel" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
      </widget>
      <widget class="CMannKeyPropertiesDlgFE" name="m_wndKeyProperties" native="true">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>300</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="CSequencerDopeSheetToolbar" name="m_cDlgToolBar" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="buttonTvJumpStart">
        <property name="icon">
         <iconset resource="../mannequindialog.qrc">
          <normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_01.png</normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_01.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonTvPlay">
        <property name="icon">
         <iconset resource="../mannequindialog.qrc">
          <normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_02.png</normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_02.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="popupMode">
         <enum>QToolButton::MenuButtonPopup</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonTvJumpEnd">
        <property name="icon">
         <iconset resource="../mannequindialog.qrc">
          <normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_03.png</normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_03.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonPlayLoop">
        <property name="icon">
         <iconset resource="../mannequindialog.qrc">
          <normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_04.png</normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_04.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonPreviewerNewSequence">
        <property name="icon">
         <iconset resource="../mannequindialog.qrc">
          <normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_05.png</normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_05.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonPreviewerSaveSequence">
        <property name="icon">
         <iconset resource="../mannequindialog.qrc">
          <normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_06.png</normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_06.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonPreviewerLoadSequence">
        <property name="icon">
         <iconset resource="../mannequindialog.qrc">
          <normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_07.png</normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_07.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonToggle1P">
        <property name="icon">
         <iconset resource="../mannequindialog.qrc">
          <normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_08.png</normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_08.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonShowDebugOptions">
        <property name="icon">
         <iconset resource="../mannequindialog.qrc">
          <normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_09.png</normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_09.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonTogglePreviewUnits">
        <property name="icon">
         <iconset resource="../mannequindialog.qrc">
          <normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_10.png</normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_10.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonMannReloadAnims">
        <property name="icon">
         <iconset resource="../mannequindialog.qrc">
          <normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_11.png</normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_11.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonMannLookAtToggle">
        <property name="icon">
         <iconset resource="../mannequindialog.qrc">
          <normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_12.png</normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_12.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonMannRotateLocator">
        <property name="icon">
         <iconset resource="../mannequindialog.qrc">
          <normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_13.png</normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_13.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonMannTranslateLocator">
        <property name="icon">
         <iconset resource="../mannequindialog.qrc">
          <normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_14.png</normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_14.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonMannAttachToEntity">
        <property name="icon">
         <iconset resource="../mannequindialog.qrc">
          <normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_15.png</normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_15.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonMannShowSceneRoots">
        <property name="icon">
         <iconset resource="../mannequindialog.qrc">
          <normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_16.png</normaloff>:/FragmentBrowser/Controls/mannequin_previewer_toolbar_16.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CMannNodesWidget</class>
   <extends>QTreeView</extends>
   <header>Mannequin/MannequinNodes.h</header>
  </customwidget>
  <customwidget>
   <class>CMannKeyPropertiesDlgFE</class>
   <extends>QWidget</extends>
   <header>Mannequin/MannKeyPropertiesDlgFE.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CSequencerDopeSheetToolbar</class>
   <extends>QWidget</extends>
   <header>Mannequin/SequencerDopeSheetToolbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReflectedPropertiesPanel</class>
   <extends>QWidget</extends>
   <header>Controls/ReflectedPropertyControl/ReflectedPropertiesPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CMannDopeSheet</class>
   <extends>QWidget</extends>
   <header>Mannequin/Controls/MannDopeSheet.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../mannequindialog.qrc"/>
 </resources>
 <connections/>
</ui>
